build.js

```diff
@@ -95,7 +95,7 @@ class WebComponentBuild {
 	}
 
 	__maakVlSrcImportsAbsoluut(file) {
-		replace(`from ${quoted('/src/vl-(.*)')}`, `from '/node_modules/vl-ui-${this.webcomponent}/dist/vl-\\$1'`, file);
+		replace(`${quoted('/src/vl-(.*)')}`, `'/node_modules/vl-ui-${this.webcomponent}/dist/vl-\\$1'`, file);
```

Comment:
Ik blijf het wat vies en gevaarlijke vinden dat we gewoon ne string replace doen ipv. een tool gebruiken die javascript snapt en op de juiste plaatsen vervangingen toepast. Het enige geluk dat we in dit geval hebben is dat ik niet zo snel "/src/vl-" zie terugkomen in source code buiten bij imports.

Reply:
We voorzien momenteel 3 builds, ik denk niet dat er een build tool bestaat die zonder aanpassing doet wat we willen dat er nu gebeurt, maar maak gerust een issue aan om dit verder te onderzoeken.

Reply:
Geen idee of er tools bestaan die al exact doen wat wij nodig hebben, maar er is inspiratie te halen uit bestaande plugins voor bv. babel (bv https://github.com/SectorLabs/babel-plugin-transform-named-imports/tree/master/src).
Ze laten de javascript door Babel parsen naar een AST en kunnen daarmee alle import statements opvragen en herschrijven.
Heb er een issue voor gemaakt: #153.